src/index.html

```diff
@@ -19,6 +19,10 @@
   <link rel="icon" href="/favicon.png" />
   <title>Register slimme apparaten</title>
 
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css"
```

Comment:
`leaflet` Is already used as a dependency in this project, why not use the CSS file from that dependency? Also, version `1.5.1` is not the same version as is used in `package.json` for `leaflet`.